### PR TITLE
DOC update build dependencies

### PR DESCRIPTION
**Pull request by NelleV**
This complements #8373 and updates the README file to reflect graphviz is also a dependency.

### CODE REVIEW

File: doc/README.txt [hunk 1]
```diff
@@ -5,9 +5,17 @@ Matplotlib documentation
 Building the documentation
 --------------------------
 
-A list of dependencies can be found in ../doc-requirements.txt.
+To build the documentation, you will need additional dependencies:
 
-All of these dependencies can be installed through pip::
+* Sphinx-1.3 or later(Version 1.5.0 is not supported)
```
Comment (anntzer) [inline doc/README.txt]: missing space, I would also drop "Version" (or at least make it "version")

File: doc/README.txt [hunk 2]
```diff
@@ -5,9 +5,17 @@ Matplotlib documentation
...
+* numpydoc 0.4 or later
+* IPython
+* Mock
```
Comment (anntzer) [inline doc/README.txt]: mock, lowercase.

File: doc/README.txt [hunk 3]
```diff
@@ -5,9 +5,17 @@ Matplotlib documentation
...
+* colorspacious
+* pillow
+* graphviz.
```
Comment (anntzer) [inline doc/README.txt]: no final dot here, or add commas after other items (e.g. first answer at http://english.stackexchange.com/questions/55048/use-of-capitals-and-full-stops-in-bullet-points)

File: doc/README.txt [hunk 4]
```diff
@@ -5,9 +5,17 @@ Matplotlib documentation
...
+
+All of these dependencies except graphviz can be installed through pip::
```
Comment (anntzer) [inline doc/README.txt]: I would emphasize (`*`) "except graphviz".

**Comment by NelleV** [comment]
Sorry, I don't have time to fix this. I'm closing this PR: this documentation problem will be perfect for a new contributor to work on.

**Comment by phobson** [comment]
@NelleV  Do you mind if I push up to this with a few changes?

**Comment by NelleV** [comment]
@phobson No, go for it. I reopened the PR and feel free to push.
I won't be able to follow during the next few days (I am at a workshop that's keeping us busy :) )

**Comment by NelleV** [comment]
I fixed an additional typo. I think it should be good.
Thanks @phobson for the help!

**Comment by anntzer** [comment]
Thanks!

**Comment by tacaswell** [comment]
backported to v2.0.0-doc as 976692fef766498f57588fabacd3e348944ada43

**Pull request merged**